index.html

```diff
@@ -0,0 +1,1008 @@
+              </h4>
+              <ul>
+                <li><del>$1839</del> $39</li>
+              </ul>
+            </div>
+          </article>
+
+          <article class="latest-deals-article">
+            <a href="#" class="deals-image-anchor deal-5"></a>
+            <div class="latest-deal-details">
+
+              <h4>
+                <a href="#">Boxne SSD Hosting Lite Plan: Lifetime Subscription</a>
+              </h4>
+              <ul>
+                <li><del>$239</del> $39</li>
+              </ul>
+            </div>
+          </article>
+
+          <article class="latest-deals-article">
+            <a href="#" class="deals-image-anchor deal-6"></a>
+            <div class="latest-deal-details">
+
+              <h4>
+                <a href="#" >The 2020 Comprehensive Programming Collection</a>
+              </h4>
+              <ul>
+                <li><del>$1659</del> $29.99</li>
+              </ul>
+            </div>
+          </article>
+
+          <article class="latest-deals-article">
+            <a href="#" class="deals-image-anchor deal-7"></a>
+            <div class="latest-deal-details">
+
+              <h4>
+                <a href="#">The Salesforce Certification Essentials Bundle</a>
+              </h4>
+              <ul>
+                <li><del>$1500</del> $39.99</li>
+              </ul>
+            </div>
+          </article>
+
+          <article class="latest-deals-article">
+            <a href="#" class="deals-image-anchor deal-8"></a>
+            <div class="latest-deal-details">
+
+              <h4>
+                <a href="#" >The Microsoft Azure 203 Certification Training Course</a>
+              </h4>
+              <ul>
+                <li><del>$399</del> $29.99</li>
+              </ul>
+            </div>
+          </article>
+          
+        </section>
+
+
+
+      </section>
+    </main>
+    <footer>
+      <div class="footer-links"> 
+        <ul class="social-links">
+          <li><a href="#"><i class="fa fa-facebook-square tar"></i></a></li>
+          <li><a href="#"><i class="fa fa-twitter tar"></i></a></li>
+          <li><a href="#"><i class="fa fa-youtube tar"></i></a></li>
+          <li><a href="#"><i class="fa fa-instagram tar"></i></a></li>
+          <li><a href="#"><i class="fa fa-pinterest-p tar"></i></a></li>
+          <li><a href="#"><i class="fa fa-rss tar"></i></a></li>
+          <li><a href="#"><i class="fa fa-envelope tar"></i></a></li>
+        </ul>
+        <ul class="site-links">
+          <li><a href="#">EVENTS</a></li>
+          <li><a href="#">ABOUT</a></li>
+          <li><a href="#">ADVERTISE</a></li>
+          <li><a href="#">JOBS</a></li>
+          <li><a href="#">CONTACT</a></li>
+        </ul>
+
+      </div>
+
+      <div class="legal-copyright">
+        <span>TNW C 2006-2020 The Next Web B.V Made with love in Amsterdam. Powered by STACKPATH</span>
+
+      </div>
```

Comment:
- [ ] Add the logos and the links to them in this section, please.

Reply:
Thanks, done